lib/ui/platform_dispatcher.dart

```diff
@@ -308,6 +308,93 @@ class PlatformDispatcher {
     _invoke(onMetricsChanged, _onMetricsChangedZone);
   }
 
+  /// A callback invoked immediately after the focus is transitioned across [FlutterView]s.
+  ///
+  /// When the platform moves the focus from one [FlutterView] to another, this
+  /// callback is invoked indicating the new view that has focus and the direction
+  /// in which focus was received. For example, if focus is moved to the [FlutterView]
+  /// with ID 2 in the forward direction (could be the result of pressing tab)
+  /// the callback receives the following [ViewFocusEvent]:
+  ///
+  /// ```dart
+  /// ViewFocusEvent(
+  ///   viewId: 2,
+  ///   state: ViewFocusState.focused,
+  ///   direction: ViewFocusDirection.forward,
+  /// )
+  /// ```
+  ///
+  /// Typically, receivers of this event respond by moving the focus to the first
+  /// focusable element inside the [FlutterView] with ID 2. If a view receives
+  /// focus in the backwards direction (could be the result of pressing shift + tab),
+  /// typically the last focusable element inside that view is focused.
+  ///
+  /// The platform can also move input focus to a non-[FlutterView] element.
```

Comment:
I know I suggested referring to "element", but I just realized that it means nothing on mobile and desktop. How about this: "The platform may remove focus from a [FlutterView]. For example, on the web, the browser can move focus to another element, or to the browser's built-in UI. On desktop, the operating system can switch to another window (e.g. using Alt + Tab on Windows)."

Reply:
Makes sense. Done